BenadrylAutomation/src/main/java/com/jnj/actions/Action.java

```diff
@@ -1004,30 +1004,16 @@ public static void navigateBack() {
 	public static void navigateForward() {
 		driver.navigate().forward();
 	}
-
+	
 	/**
 	 * Function to verify page url
```

Comment:
@zahinSJI there should be empty line after this. i can see you have removed that, pls add again